src/homework2/task2.py

```diff
@@ -10,14 +12,25 @@ def longest_word(str_):
     :param str_: входная строка
     :return: строка. Самое длинное слово в предложении (в случае если их
         несколько, самое левое в строке).
-        в случае если
     """
-
     # write your code here
-    return ''  # write return value here
+    str_1 = ''
+    for char in str_:  # убираем все не буквы и не пробелы
```

Comment:
Appolo13 wouldn't pass it

Reply:
we shouldn't ignore numbers

Reply:
I was in doubt while creating first version of the function whether I should ignore numbers or not because a) I couldn't imagine at a glance words together with numbers and b) counting numbers wasn't explicitly indicated in the task description. Now I see that there are some words including numbers like 'b2b', '4U', 'U2', hashes etc. But for the record, 'Apollo 13' is usually written as two distinct words.